bom/runtime/pom.xml

```diff
@@ -1049,7 +1049,7 @@
             </dependency>
             <dependency>
                 <groupId>io.smallrye</groupId>
-                <artifactId>smallrye-fault-tolerance</artifactId>
+                <artifactId>smallrye-fault-tolerance-2.0</artifactId>
```

Comment:
This should be a separate PR that we should integrate separately so that it's traceable.

Not saying you should do it, considering you're on PTO soon but we can do it and rebase this PR on top.

Reply:
Oh, unless the old SmallRye Metrics won't work with SmallRye FT 2.0?

In this case, it makes sense to have only one PR.

Reply:
Yes, it wouldn't be possible to split it into two in a way that they would work separately